app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.user_mailer.task_reminder.subject
+  #
+  def task_reminder(user)
+    @greeting = "Hi"
+    @user = user
+    @todo_list = user.todo_lists.where('deadline < ?', Time.now + 1.day).where('deadline > ?', Time.now)
```

Comment:
Line is too long. [104/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.